grpcgcp/grpc_gcp/grpc_gcp.proto

```diff
@@ -41,7 +46,7 @@ message MethodConfig {
   // sequentially, and the first one takes precedence.
   repeated string name = 1;
 
-  // The channel affinity configurations.
+  // The channel/connection affinity configurations.
```

Comment:
I think keeping the terminology as "channel" here and elsewhere is OK and probably good. Your note at the top of doc.go is sufficient to explain what's going on, in my opinion.

Reply:
Done.